tests/koreo_tooling/indexing/test_koreo_semantics.py

```diff
@@ -1,7 +1,5 @@
 import unittest
 
-from koreo_tooling.indexing import koreo_semantics
```

Comment:
Whoops. This file appears to be ... empty. We should leave the import since at least that would catch super egregious syntax issues. :)